src/tests/examples/code_generator.py

```diff
@@ -61,7 +61,14 @@
     '072 Hydro Reservoir Model -enhanced-05',
     '073 Hydro Reservoir Model -enhanced-06',
     '074 KCG on Four areas -01',
-    '075 KCG on Four areas -02']
+    '075 KCG on Four areas -02',
+    'playlist-0',
```

Comment:
what are these tests ?

Reply:
these are the tests provided by Michel Doquet to check values in case of MC Scenario Playlist weight definition